tests/conftest.py

```diff
@@ -11,15 +11,15 @@
 from tests.integration_tests import ADMIN_TOKEN
 
 
-@pytest.yield_fixture(scope="session")
+@pytest.fixture(scope="session")
```

Comment:
Is it really correct to change here? We are yielding and tearing down in tests.

Reply:
The yield_fixture is deprecated according to the docs and does the same as fixture:
https://docs.pytest.org/en/latest/yieldfixture.html

Reply:
Nice, didn't know that, thanks!